diff --git a/node_modules/react-native-autocomplete-dropdown/src/NothingFound.tsx b/node_modules/react-native-autocomplete-dropdown/src/NothingFound.tsx
index c128f15..23d691a 100644
--- a/node_modules/react-native-autocomplete-dropdown/src/NothingFound.tsx
+++ b/node_modules/react-native-autocomplete-dropdown/src/NothingFound.tsx
@@ -23,5 +23,5 @@ const styles = StyleSheet.create({
   container: {
     padding: 10,
   },
-  text: { textAlign: 'center' },
+  text: { textAlign: 'center' , color:'black' },
 })
